<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;0.&lt;/span&gt;选择设备:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cBoxDevice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteAll">
       <property name="text">
        <string>烧写全部文件(1~4)</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;1.&lt;/span&gt;Uboot:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Uboot"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbtnSelectUboot">
       <property name="toolTip">
        <string>选择文件.</string>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteUboot">
       <property name="toolTip">
        <string>烧写Uboot</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>烧写</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;2.&lt;/span&gt;设备树:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Tree"/>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnSelectTree">
       <property name="toolTip">
        <string>选择文件.</string>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteTree">
       <property name="toolTip">
        <string>烧写设备树</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>烧写</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;3.&lt;/span&gt;内核:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Kernel"/>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnSelectKernel">
       <property name="toolTip">
        <string>选择文件.</string>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteKernel">
       <property name="toolTip">
        <string>烧写内核</string>
       </property>
       <property name="text">
        <string>烧写</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;4.&lt;/span&gt;文件系统:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Rootfs"/>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnSelectRootfs">
       <property name="toolTip">
        <string>选择文件.</string>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteRootfs">
       <property name="toolTip">
        <string>烧写文件系统</string>
       </property>
       <property name="text">
        <string>烧写</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;5.&lt;/span&gt;裸机文件:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Single"/>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnSelectSingle">
       <property name="toolTip">
        <string>选择文件.</string>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteSingle">
       <property name="toolTip">
        <string>烧写裸机文件</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>烧写</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;6.&lt;/span&gt;用户文件:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_User"/>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnSelectUser">
       <property name="toolTip">
        <string>选择文件.</string>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/选择.png</normaloff>:/res/选择.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnWriteUser">
       <property name="toolTip">
        <string>上传用户文件</string>
       </property>
       <property name="text">
        <string>上传</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/res/下载.png</normaloff>:/res/下载.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
